src/api.js

```diff
@@ -0,0 +1,15 @@
+var express = require('express');
+var app = express();
+
+app.get('/stub', function(req, res) {
+    res.redirect('http://31.media.tumblr.com/65f2c221257116959599522008abfa4c/tumblr_n7qalc0aPr1r7tu05o2_500.gif');

```

Comment:
1. Should all constants or hard coded urls be in one JS file to create a 
2. This class would then use the constant instead of a direct ref to promote maintainability, re-usability and separation of concerns.